src/App.tsx

```diff
@@ -1,61 +1,108 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import React, { useState } from 'react';
+import { Todo } from './types/Todo';
+import { Select } from './components/Select/Select';
+import { Input } from './components/Input/Input';
+import { User } from './types/User';
+
+const getTodoList = todosFromServer.map(todo => ({
+  ...todo,
+  user: usersFromServer.find(u => u.id === todo.userId) || {},
+}));
```

Comment:
The `getTodoList` function assigns an empty object as the user if no matching user is found. This could lead to issues if the rest of the application expects a `User` object. Consider handling this case more explicitly, perhaps by filtering out todos without a valid user.